keras_retinanet/models/retinanet.py

```diff
@@ -181,11 +181,17 @@ def retinanet(
     anchor_parameters       = AnchorParameters.default,
     create_pyramid_features = __create_pyramid_features,
     submodels               = None,
+    freeze_backbone         = False,
```

Comment:
I'm wondering if adding another parameter here is the smart thing to do. The added parameter doesn't make the function more flexible. The backbone could have been frozen before passing it to this function instead.

If it is currently to cumbersome to freeze the whole backbone, maybe we could add a utility function to freeze an entire model.

For the `resnet` functions we could add callback to modify the backend model before passing it down to `retinanet` rather than specifically adding a `freeze_backend` option. That would be more future proof (allow more modifications without exploding the number of arguments).

Reply:
I see your point. If I have the time for it I'll implement this, but it could be that I postpone this a bit.